### ISSUE TITLE
Ars Nouveau items appear faded on hotbar

### ISSUE THREAD

**Issue by Swedz** [RaphiMC/ImmediatelyFast#133]
Items with 3d models from Ars Nouveau (spell book, for example) appear faded when on the hotbar. I have not noticed this with other mods, but it only happens when both Ars Nouveau and ImmediatelyFast are installed.

**Comment by RaphiMC** [comment]
Provide screenshots, a game log and detailed steps to reproduce

**Comment by Swedz** [comment]
Screenshot: ![image](https://github.com/RaphiMC/ImmediatelyFast/assets/6707125/e5da4d7c-4923-4b2c-a76e-89ac3fbacf57)
Log: [2023-10-25-1.log](https://github.com/RaphiMC/ImmediatelyFast/files/13163303/2023-10-25-1.log)

Steps to reproduce:
1. Install Ars Nouveau, Curios (requirement for Ars), and ImmediatelyFast
2. Place the Ars Nouveau Novice Spell Book on your hotbar

Hope this helps!

**Comment by genesy** [comment]
Yup happens to 3d models only on the hud. in inventory it renders fine.

**Comment by SiverDX** [comment]
Same issue here https://github.com/Sonmok14/FromTheShadows/issues/14

**Comment by KakashoLin** [comment]
Still an issue today, currently disabling Immediately Fast in my modpack till this issue is resolved.

**Comment by Overtekk** [comment]
Can confirm this issue. It also do that with create: teleporters. 
With a lot of mods, all 3d items will rendered faded when you put spell book of ars nouveau or create: teleporters 3d item (didn't test with all items).

**Comment by KakashoLin** [comment]
This also happens with any Tomes of Blood

**Comment by KakashoLin** [comment]
Issue referenced here: https://github.com/embeddedt/embeddium/issues/75

**Comment by txnitxnichopper** [comment]
@RaphiMC Don't think anyone has mentioned it yet, but I'm pretty sure this is happening with all Geckolib/Azurelib item models. This bug also happens with my Create addon that uses it.

**Comment by RaphiMC** [comment]
> @RaphiMC Don't think anyone has mentioned it yet, but I'm pretty sure this is happening with all Geckolib/Azurelib item models. This bug also happens with my Create addon that uses it.

What version of geckolib are you using? The issue has been fixed on their side

**Comment by RaphiMC** [comment]
Closed due to inactivity

**Comment by As3r1408** [comment]
try using this settings in the config file:

{
  "REGULAR_INFO": "----- Regular config values below -----",
  "font_atlas_resizing": true,
  "map_atlas_generation": true,
  "hud_batching": false,
  "fast_text_lookup": true,
  "fast_buffer_upload": true,
  "COSMETIC_INFO": "----- Cosmetic only config values below (Does not optimize anything) -----",
  "dont_add_info_into_debug_hud": false,
  "EXPERIMENTAL_INFO": "----- Experimental config values below (Rendering glitches may occur) -----",
  "experimental_disable_error_checking": false,
  "experimental_disable_resource_pack_conflict_handling": false,
  "experimental_sign_text_buffering": false,
  "experimental_screen_batching": false,
  "DEBUG_INFO": "----- Debug only config values below (Do not touch) -----",
  "debug_only_and_not_recommended_disable_universal_batching": false,
  "debug_only_and_not_recommended_disable_mod_conflict_handling": false,
  "debug_only_and_not_recommended_disable_hardware_conflict_handling": false,
  "debug_only_print_additional_error_information": false,
  "debug_only_use_last_usage_for_batch_ordering": false
}

**Comment by edralzar** [comment]
I had both mods installed for a little while without issue, but this popped up since Jan 25 (latest update). With a resource pack, the items in the hotbar are fully transparent. Without a resource pack they are faded out when holding the 3d item (Ars Nouveau Spellbook).

I can also confirm that setting the `hud_batching` setting to `false` fixes the issue.

**Comment by RaphiMC** [comment]
Upload the game log and a list of mods including their version

**Comment by edralzar** [comment]
The log doesn't appear to contain anything, but here it is: 
[latest.log](https://github.com/user-attachments/files/18604522/latest.log)


List of mods is:

[Ars Nouveau](http://tauri.localhost/project/TKB6INcv?i=Forge-Test)
[by baileyholl](https://modrinth.com/user/baileyholl)
4.9.0
ars_nouveau-1.20.1-4.9.0-all.jar



[Curios API](http://tauri.localhost/project/vvuO3ImH?i=Forge-Test)
[by TheIllusiveC4](https://modrinth.com/user/TheIllusiveC4)
5.11.1+1.20.1
curios-forge-5.11.1+1.20.1.jar



[ImmediatelyFast](http://tauri.localhost/project/5ZwdcRci?i=Forge-Test)
[by RaphiMC](https://modrinth.com/user/RaphiMC)
1.3.4+1.20.4-forge
ImmediatelyFast-Forge-1.3.4+1.20.4.jar



[Nautilus 3D](http://tauri.localhost/project/1MyOrwfu?i=Forge-Test)
[by FabianMPunkt](https://modrinth.com/user/FabianMPunkt)
V1.9
Nautilus3D_V1.9_[MC-1.13+].zip

**Comment by Tabulika** [comment]
Setting "hud_batching" to false actually fixes this problem with geckolib's 3d stuff

**Comment by RaphiMC** [comment]
I fixed this issue in the latest dev build: https://github.com/RaphiMC/ImmediatelyFast/actions/runs/13641780438
